### PR TITLE
[WIP, don't merge] use search API to determine relevant PRs

### PR DESCRIPTION
**Pull request by lewurm**

### CODE REVIEW

File: monors.py [hunk 1]
```diff
@@ -247,6 +247,8 @@ def main():
     pulls = gh.repos (cfg["owner"]) (cfg["repo"]).pulls ().get (state="open", sort="updated", direction="desc", per_page = 100)
     logging.info("found %d pull requests", len (pulls))
 
+    # We have a limit on the number of HTTP request to the github API (5000/h), each PR takes 3+ request, every 5 minutes.
+    # With 25, we arrived to ~2000/h requests per hour, so with 50, we should arrive at ~4000/h.
```
Comment (akoeplinger) [inline monors.py]: Can we instead just process the PRs that have "@monojenkins merge"? We should be able to do this via a simple search, like https://api.github.com/search/issues?q=is:open+repo:mono/mono+%22@monojenkins%20merge%22&type=pr

Comment (luhenry) [inline monors.py]: we could broaden that search to `@monojenkins` but otherwise I agree

**Comment by lewurm** [comment]
I tried to implement @akoeplinger suggestion, but it doesn't integrate nicely with the github API bindings we use... @ludovic-henry what do you think?

**Comment by akoeplinger** [comment]
We didn't really hit an issue with the current implementation and the search API approach won't work with the Slack notification changes anyway, so closing :)

**Pull request closed without merging**